### ISSUE TITLE
Is the accuracy reported in tables 1, 2, and 3 from the best epoch or the last epoch?

### ISSUE THREAD

**Issue by guixianjin** [YisenWang/symmetric_cross_entropy_for_noisy_labels#8]
Dear authors,
  Thanks for your interesting work.
  Since we usually train neural network many epochs, and after each epoch's training, we may test the neural network and get a test accuracy. Usually, the highest test accuracy may occur in some epoch of the intermediate training process, not necessarily the last epoch.

In your paper, is the accuracy reported in tables 1, 2, and 3 from the best epoch or the last epoch?

**Comment by yanghao1126** [comment]
Hi,

I am also confused about this question.

**Comment by YisenWang** [comment]
Hi all, thanks for your interest in our work. The accuracies in the tables are the last epoch.
